lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.t

```diff
@@ -36,7 +36,7 @@ symlink($reference_fasta_index,$reference_build->data_directory .'/all_sequences
 
 my $result_users = Genome::Test::Factory::SoftwareResult::User->setup_user_hash(reference_sequence_build_id => $reference_build->id,);
 
-my $params = "-g,-d,-o:Hello";
+my $params = '-P,-g,-d';
```

Comment:
What's the reasoning behind removing the `-o:Hello` test case? It would be great to still have a test for that syntax.